pkg/arvo/sys/hoon.hoon

```diff
@@ -8764,6 +8764,7 @@
       %peek   peek
       %repo   repo
       %rest   rest
+      %sink   sink
```

Comment:
For future reference, this hook should not have been added in this PR. It changes the jet-matching interface (per hashboard at least), and so requires staging on both sides.